src/consensus.rs

```diff
@@ -44,12 +42,13 @@ impl Consensus {
         bootstrap_peer: Option<Uri>,
         uri: Option<String>,
         p2p_port: Option<u32>,
+        config: ConsensusConfig,
     ) -> anyhow::Result<(Self, SyncSender<Message>)> {
-        let config = Config {
+        let raft_config = Config {
```

Comment:
renamed to `raft_config` to free the `config` binding for the `ConsensusConfig`

Reply:
Maybe import the type as `RaftConfig`?